doc/classes/DampedSpringJoint2D.xml

```diff
@@ -78,12 +78,18 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			The spring joint's damping ratio.
+			When the two bodies move into different directions the system tries to allign them to the spring axis again.
```

Comment:
3 lines = 3 paragraphs in the docs. One l for align and alignment. Suggestion:

The spring joint's damping ratio. A value between [code]0[/code] and [code]1[/code]. When the two bodies move into different directions the system tries to align them to the spring axis again. A high [code]damping[/code] value forces the attached bodies to align faster. Default value: [code]add default[/code]